doc/man3/X509_check_host.pod

```diff
@@ -62,7 +62,8 @@ X509_check_ip() checks if the certificate matches a specified IPv4 or
 IPv6 address.  The B<address> array is in binary format, in network
 byte order.  The length is either 4 (IPv4) or 16 (IPv6).  Only
 explicitly marked addresses in the certificates are considered; IP
-addresses stored in DNS names and Common Names are ignored.
+addresses stored in DNS names and Common Names are ignored. The
+B<flags> argument currently has no effect.
```

Comment:
Maybe `There are currently no B<flags> that would affect the behavior of this call.`

Reply:
@t8m Updated as suggested! :)